src/pages/build-an-app.md

```diff
@@ -83,10 +83,9 @@ on Blockstack are written in the [Clarity language](https://clarity-lang.org). V
 [@page-reference | inline]
 | /smart-contracts/public-registry-tutorial
 
-### Connect
+### Stacks.js
 
-Connect is a JavaScript library developed by Blockstack PBC that makes it easy to integrate authentication, data storage
+stacks.js is a collection of JavaScript library developed by Blockstack PBC that makes it easy to integrate authentication, data storage
 and smart contracts functionality in a user-friendly way.
 
-[@page-reference | inline]
-| /authentication/connect
+[Stacks.js](https://blockstack.github.io/stacks.js/)
```

Comment:
As opposed to a new line with just the link to the references, could you link the first `stacks.js` in the paragraph above?

Reply:
@agraebe  I don't understand where the link should point to.